### PR TITLE
Raise 'mouseenter' and 'mouseleave' events during moving (closes #1426)

### LINKED ISSUE #1426: EventListeners 'mouseenter' and 'mouseleave' not working with testcafe
### Are you requesting a feature or reporting a bug?

Bug

### What is the current behavior?

I have a button that should show up on 'mouseenter' and disappear on 'mouseleave', but nothing shows when running tests. Test passes anyway since element with opacity = 0 is clicked.

### What is the expected behavior?

hidden element should show up

### How would you reproduce the current behavior (if this is a bug)?

when I reproduce this manually everything works fine.
1. go to URL given below
2. hover over red square
3. blue element with button should appear but id doesn't
4. click button to change red to yellow

#### Provide the test code and the tested page URL (if applicable)

Tested page URL: https://jsfiddle.net/p5reyep3/3/

Test code

```js
import { Selector } from 'testcafe';

fixture('click a thing on hover')
.page('https://jsfiddle.net/p5reyep3/3/');

test('succeed to click', async (t) => {
  await t
  .switchToIframe('[name="result"]')
  .hover('#outer')
  .click('button')
  .expect(Selector('#outer').hasClass('yellow')).eql(true);
});

```

### Specify your

* operating system: macOS Sierra 10.12.4
* testcafe version: 0.14.0
* node.js version: 6.10.0

### PR DESCRIPTION
**Pull request by AlexanderMoskovkin**
[will be refactored when #1490 is merged]

### CODE REVIEW

File: src/client/automation/playback/move/index.js [hunk 1]
```diff
@@ -189,8 +189,10 @@ export default class MoveAutomation {
             var clientY = startY - iframeRectangle.top;
 
             // NOTE: We should not emulate mouseout if iframe was reloaded.
```
Comment (AndreyBelym) [inline src/client/automation/playback/move/index.js]: NOTE: We should not emulate mouseout and mouseleave if iframe was reloaded.

**Comment by AlexanderMoskovkin** [comment]
FPR

**Comment by AlexanderMoskovkin** [comment]
Just an unstable test

**Pull request merged**